tpm2/open_other.go

```diff
@@ -17,26 +17,41 @@
 package tpm2
 
 import (
+	"errors"
 	"fmt"
 	"io"
+	"os"
 
 	"github.com/google/go-tpm/tpmutil"
 )
 
 // OpenTPM opens a channel to the TPM at the given path. If the file is a
 // device, then it treats it like a normal TPM device, and if the file is a
 // Unix domain socket, then it opens a connection to the socket.
-func OpenTPM(path string) (io.ReadWriteCloser, error) {
-	rwc, err := tpmutil.OpenTPM(path)
+//
+// This function may also be invoked with no paths, as tpm2.OpenTPM(). In this
+// case, the default paths on Linux (/dev/tpmrm0 and /dev/tpm0), will be used.
+func OpenTPM(path ...string) (tpm io.ReadWriteCloser, err error) {
+	switch len(path) {
+	case 0:
+		tpm, err = tpmutil.OpenTPM("/dev/tpmrm0")
+		if os.IsNotExist(err) {
```

Comment:
we should use `errors.Is(err, os.ErrNotExist)` https://golang.org/src/os/error.go?s=3221:3252#L82